src/CHARTER.md

```diff
@@ -0,0 +1,52 @@
+# wg-async-foundations charter
+<!--
+ Provide an introduction summarising the goals and motivation behind your
+ project group.
+-->
+
+## Goals
+
+<!--
+ Explain what changes you'd like to see your group your group to focus on, and
+ how you plan to approach these issues. Focus on explaining the highest possible
+ level of your change.
+-->
+
+This working group is focused around implementation/design of the “foundations” for Async I/O. This means that we are focused on designing and implementing extensions to the language, standard library, and other "core" bits of support offered by the Rust organization.
+
+## Constraints And Considerations
+
+We do not directly work on external projects like [tokio], [async-std], [smol], [embassy] and so forth, although we definitely discuss ideas and coordinate with them where appropriate.
+
+[tokio]: https://tokio.rs/
+[async-std]: https://async.rs/
+[smol]: https://github.com/smol-rs/smol/
+[embassy]: https://github.com/akiles/embassy
+
+
+<!--
+ Explain the scope of your group, what you have chosen not to include in
+ your goals, and your motivations behind making them non-goals.
+-->
+
+## Membership
+
+Leads: [@tmandry] and [@nikomatsakis]
+
+[Members]
+
+[@tmandry]: https://github.com/tmandry
+[@nikomatsakis]: https://github.com/nikomatsakis
+[Members]: https://www.rust-lang.org/governance/wgs/wg-async-foundations
+
+### Membership requirements
+
+Members are invited at the discretion of the working group leads, usually after a period of sustained contribution to the working group.
+
+In order to remain an active member, WG members must have active participation in last 6 months, where participation is one of the following:
+
+* Attended triage/sprint meetings
+* Opened PRs
+* Reviewed PRs
```

Comment:
Presumably these PRs should be async-related PRs in either this repo, or in rust-lang/rust?   Would PRs in a repo like tokio be applicable here?

Reply:
I would think so. Tokio is an independent project.

Reply:
clarified